frontend/src/routes/ServicesQueues.js

```diff
@@ -1,87 +1,141 @@
-import React, { useState } from "react"
-import PropTypes from "prop-types"
+import React from "react"
+import clsx from "clsx"
+import { makeStyles, useTheme } from "@material-ui/core/styles"
+import Drawer from "@material-ui/core/Drawer"
+import CssBaseline from "@material-ui/core/CssBaseline"
+import AppBar from "@material-ui/core/AppBar"
+import Toolbar from "@material-ui/core/Toolbar"
+import Divider from "@material-ui/core/Divider"
+import IconButton from "@material-ui/core/IconButton"
+import MenuIcon from "@material-ui/icons/Menu"
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft"
+import ChevronRightIcon from "@material-ui/icons/ChevronRight"
+import Navbar from "../components/Navbar"
+import UserSearch from "../components/UserSearch"
 import AllQueues from "../components/AllQueues"
-import ParticipantSearch from "../components/ParticipantSearch"
-import {
-  Grid,
-  Paper,
-  IconButton,
-  Typography,
-  Drawer,
-  Hidden,
-} from "@material-ui/core"
-import { makeStyles } from "@material-ui/styles"
-import SearchIcon from "@material-ui/icons/Search"
 
-const drawerWidth = 240
+const drawerWidth = 300
 
 const useStyles = makeStyles(theme => ({
   root: {
     display: "flex",
   },
-  searchButton: {
-    marginRight: 20,
-    [theme.breakpoints.up("lg")]: {
-      display: "none",
-    },
+  appBar: {
+    transition: theme.transitions.create(["margin", "width"], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+  },
+  appBarShift: {
+    width: `calc(100% - ${drawerWidth}px)`,
+    marginLeft: drawerWidth,
+    transition: theme.transitions.create(["margin", "width"], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  hide: {
+    display: "none",
   },
   drawer: {
     width: drawerWidth,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+    /* background: "#F2FCFF", */
```

Comment:
Should this be `background: theme.background.default` now?

Reply:
I defer to you. I assumed the purpose of moving the background color into the theme was so it would be present by default.  Is that not how it works?

Reply:
It throws an error if I do that. Cannot red property 'default' of undefined.

Reply:
Pushed the other change.  Maybe background: theme.palette.background.default?